src/lang-rust.js

```diff
@@ -0,0 +1,79 @@
+// Copyright (C) 2015 Chris Morgan
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+/**
+ * @fileoverview
+ * Registers a language handler for Rust.
+ *
+ * Derived from prior experience implementing similar things in a few environments,
+ * most especially rust.vim.
+ *
+ * @author me@chrismorgan.info
+ */
+
+PR['registerLangHandler'](
+    PR['createSimpleLexer']([], [
+		// Whitespace
+		[PR['PR_PLAIN'],       /^[\t\n\r \xA0]+/],
+
+		// Single line comments
+		[PR['PR_COMMENT'], /^\/\/.*/],
+		// Block comments (sadly I do not see how to make this cope with comment nesting as it should)
```

Comment:
Yep.   Sorry.  I haven't had time to relax the regular-lexical-grammar assumptions, and there's no workaround for Rust&Ocaml's nesting comments the way there is for Lua.